pt-BR/mail/chrome/communicator/utilityOverlay.properties

```diff
@@ -1,7 +1,6 @@
 
-onlineTooltip0=You are online (proxy: none). Click the icon to go offline.
-onlineTooltip1=You are online (proxy: manual). Click the icon to go offline.
-onlineTooltip2=You are online (proxy: auto URL). Click the icon to go offline.
-onlineTooltip4=You are online (proxy: auto discover). Click the icon to go offline.
-offlineTooltip=You are offline. Click the icon to go online.
-
+onlineTooltip0=Você está online (proxy: nenhum). Clique no ícone para ficar offline.
+onlineTooltip1=Você está online (proxy: manual). Clique no ícone para ficar offline.
+onlineTooltip2=Você está online (proxy: URL automática). Clique no ícone para ficar offline.
```

Comment:
CORRECTIONS NEEDED it should be onlineTooltip2=Você está online (proxy: URL automático). Clique no ícone para ficar offline.